honeybee_vtk/writers.py

```diff
@@ -1,20 +1,26 @@
-"""A collection of helper functions to write VTK to file."""
+"""A collection of helper functions to write VTK objects to file."""
 
 import vtk
 import os
 
 from typing import List
-from .to_vtk import create_polygons, point_vectors, create_arrows
-from .helper import get_end_point, get_vector_at_center
+from .to_vtk import create_polygons, create_points, create_arrows
+from .helper import get_end_point, get_vector_at_center, get_point3d, get_vector3d
 from .hbjson import get_grid_base, get_grid_mesh, get_grid_points
 
 
 def write_polydata(
-        polydata, file_name, vtk_writer, vtk_extension, target_folder):
+        grouped_points, file_name, vtk_writer, vtk_extension, target_folder):
     """Write VTK Polydata to a file.
 
     Args:
-        polydata: An appended VTK polydata object.
+        grouped_points: A dictionary with Honeybee type as keys and list of 
+        lists of Point3Ds for geometry that belongs to that Honeybee type. 
+        An example would be;
+        {
+        'Wall': [[Point1, Point2, Point3], [Point4, Point5, Point6, Point7]],
+        'Aperture': [[Point1, Point2, Point3], [Point4, Point5, Point6, Point7]]
+        }
```

Comment:
for this to be rendered correctly you should use `.. code-block:: python`. Here is an example: 

https://github.com/ladybug-tools/ladybug/blob/3249254347ef466be0aa9b97c86977072ae0a93e/ladybug/dt.py#L65-L72